### PR TITLE
feat(ui): C# issue detail improvements

### PR DESCRIPTION
**Pull request by matejminar**
- less cluttered stacktrace
- additional info moved to togglable space under frame
- packages renamed to assemblies 
- packages(assemblies) have more space for name and less for version
- breadcrumbs redistributed spacing, title trimmed from left, because their beginning is often the same

**Before:**
Untitled

**After:**
image
image
image

### CODE REVIEW

**Review by bruno-garcia — APPROVED** [review]
❤️

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  version,
+  culture,
+  publicKeyToken,
+  filePath,
+}) => {
+  return (
+    <AssemblyWrapper>
+      <Icon src="icon-return-key" />
+      <AssemblyInfo>
+        <Caption>Assembly:</Caption>
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: Should these labels/captions be wrapped in `t()`?

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <FilePathInfo>
+          <Caption>Path:</Caption>
+          <TextCopyInput>{filePath}</TextCopyInput>
+        </FilePathInfo>
+      )}
+    </AssemblyWrapper>
+  );
+};
+
+// TODO: we should be able to delete these after disabling react/prop-types rule in tsx functional components
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]:
```suggestion
// TODO(ts): we should be able to delete these after disabling react/prop-types rule in tsx functional components
```

The `ts` suffix will help future us find these.

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+};
+
+const AssemblyWrapper = styled('div')`
+  font-size: 80%;
+  display: flex;
+  flex-wrap: wrap;
+  color: ${p => p.theme.gray5};
+  text-align: center;
+  position: relative;
+  padding: 0 20px 0 50px;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: Should these use `space()` to generate the value?

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx [hunk 4]
```diff
@@ -29,11 +31,19 @@ class CrumbTable extends React.Component {
...
+
     return (
       <table className="table key-value">
         <thead>
           <tr>
-            <td className="key">
+            <td
+              className="key"
+              css={
+                widerCategory && {maxWidth: '215px !important', width: '215px !important'}
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx]: Does 215 need to change in smaller viewports?

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx (hunk 1 above)
Comment (bruno-garcia) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: I'd say an assembly is an assembly in any language.

Here are Microsoft docs in Portuguese 

https://docs.microsoft.com/pt-br/dotnet/standard/assembly/

Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: Wrapped them in t`()` just to be sure.

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: thanks, fixed and will keep that in mind in the future

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx (hunk 3 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: ok

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx (hunk 4 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx]: thanks, changed this to styled component with media query 👍

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx (hunk 1 above)
Comment (bruno-garcia) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: 😢

Comment (bruno-garcia) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: image
I'd say we don't add this, if this gets translate it'll be super confusing.

Comment (billyvg) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: fwiw we haven't updated our localization in a long time, it doesn't get auto translated, I think we should get in the habit of always using `t()` for when we do decide to fix our localization though.

Comment (bruno-garcia) [reply src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx]: Fair enough. I'm just hoping we don't put it here (maybe needs a comment to clarify it's intentional?) this English word is the name of the thing in any language. If it translates it won't make sense.

**Pull request merged**